org.scala-ide.sdt.core/src/org/scalaide/core/internal/statistics/Statistics.scala

```diff
@@ -16,9 +21,12 @@ import Features._
 /**
  * Contains all definitions that belong to the statistics tracker.
  */
-class Statistics {
+class Statistics extends AnyRef with HasLogger {
```

Comment:
No hidden feature. I just don't like to use `extend` with a trait. Unnecessary thoughts...